kolibri/plugins/facility/assets/src/views/users/common/UsersTable.vue

```diff
@@ -770,4 +790,13 @@
     min-height: 0;
   }
 
+  // Vertically center KCircularLoader within the table
+  /deep/ .k-table-wrapper > p {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin: 0;
+    transform: translate(-50%, -50%);
+  }
```

Comment:
Should this be a KDS issue instead? 👀

Reply:
Yes, I do think centering the circle loader should be taken care of in a KDS issue/pr. Partially because I'm not sure the loader should be wrapped in a paragraph tag and might eventually be removed.